README.md

```
@@ -609,6 +610,15 @@ The promotion threshold (how many distinct values at a path position before it's
 katana -u https://tesla.com -fsu -fst 5
 ```
 
+*`-max-domain-pages`*
```

markdownlint-cli2 (0.22.0)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)